.storybook/main.js

```diff
@@ -1,9 +1,9 @@
 const path = require('path');
 
 module.exports = {
-  stories: ['../src/**/*.stories.@(js|mdx)', '../guidelines/**/*.stories.@(js|mdx)'],
-  addons: ['@storybook/addon-postcss', '@storybook/addon-essentials', '@storybook/addon-a11y'],
-  webpackFinal: async (config) => {
+  stories: ['../src/**/*.stories.@(js|ts|mdx|tsx)', '../guidelines/**/*.stories.@(js|ts|mdx|tsx)'],
```

Comment:
This is reminding me that we never had a conversation about `.tsx`. [Some people don't like it](https://github.com/airbnb/javascript/pull/985#issuecomment-239145468). We currently have mixed implementations between our repos and I wonder if now is a good time to try to standardize it and get it into a style guide.

Reply:
Discussed offline. We want to move to using strictly js & ts.

Reply:
Followed up further offline, there are some limitations to the ts compiler at the moment that force us to use `.tsx`.